src/pages/jonghyuk/Main/Comment.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { useState } from 'react';
+import Newcomment from './Newcomment';
+
+const Comment = () => {
+  const [comment, setComment] = useState('');
+  const hendleComment = event => {
+    setComment(event.target.value);
+  };
+  const [commentArray, setCommentArray] = useState([]);
+  const eventComment = () => {
+    const result = [...commentArray, comment];
+    setCommentArray(result);
+    setComment('');
+  };
+
+  const changeColor = comment.length > 0;
+
+  return (
+    <div className="comment">
+      <div className="like">
+        <img
+          className="commentImg"
+          alt="commentimg"
+          src="images/jonghyuk/feedimg.jpg"
+        />
+        <span className="likeEach">
+          <b>aineworld</b>님 <b>외 10명</b>이 좋아합니다
+        </span>
+      </div>
+
+      <div className="post">
+        <div>
+          <p>
+            <b>canon_mj</b> 위워크에서 진행한 베이킹 클래스...
+          </p>
+        </div>
+        <input className="moreBtn" type="button" value="더 보기" />
+      </div>
+
+      <div className="mention">
+        <div>
+          <b>neceosecius</b> 거봐 좋았잖아~~~~
+        </div>
+        <img className="like" alt="like" src="images/jonghyuk/heart.png" />
+      </div>
+      <div className="lastHour">42분 전</div>
+      <ul className="comment-ul">
+        {commentArray.map(function (a, i) {
+          return <Newcomment key={i} comment={a} />;
+        })}
```

Comment:
아쉬운 부분이 2가지 있습니다.
1. key에 index를 부여 한 점
2. a라는 변수명

1번에 대한 부연 설명 
key에 인덱스를 사용하는 것은 권장하지 않습니다
[공식문서](https://ko.reactjs.org/docs/lists-and-keys.html#embedding-map-in-jsx)
[참고자료](https://robinpokorny.medium.com/index-as-a-key-is-an-anti-pattern-e0349aece318)

2번에 대한 부연 설명
how about 
```suggestion
        {commentArray.map((comment, i) => <Newcomment key={i} comment={comment} /> )}
```

Reply:
네 감사합니다 수정하였습니다!